release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1780,7 +1761,7 @@ With this update, the CMO now properly propagates the external labels that you c
 
 * Previously, pod failures were artificially extending the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and the certificates are correctly rotated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020484[*BZ#2020484*])
 
-* In {product-title} {product-version} the default cluster-wide pod security admission policy is set to `baseline` for all namespaces and the default warning level is set to `restricted`. Before this update, Operator Lifecycle Manager displayed pod security admission warnings in the `operator-marketplace` namespace. With this fix, reducing the warning level to `baseline` resolves the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088541[*BZ#2088541*])
+* In {product-title} {product-version} the default cluster-wide pod security admission policy is set to `privileged` for all namespaces and the default warning level is set to `restricted`. Before this update, Operator Lifecycle Manager displayed pod security admission warnings in the `operator-marketplace` namespace. With this fix, reducing the warning level to `baseline` resolves the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088541[*BZ#2088541*])
```

Comment:
> Before this update,

How to understand it? What's this `update` means? The PSA? If yes, should be `after this update`.
> warnings in the `operator-marketplace` namespace.

And, should be `openshift-marketplace`, not the `operator-marketplace`.

Reply:
I changed the namespace to `openshift-marketplace`. 

Before this update refers to BZ2088541. This is just a bug fix note for the doc text supplied in the BZ. Though I have cleaned it up to conform with our style guide. I noticed that the doc text incorrectly said that baseline was the default, so I was fixing it here.

Reply:
"Before this update" is referring to the time before the bug was fixed. "With this fix" is referring to the time after the BZ was fixed.